cjaasBrowserSDK/src/interrupter/interrupter.ts

```diff
@@ -70,7 +70,7 @@ export class CJaaSInterrupter {
   listenToStream() {
     // signature in SAS Token needs to be URL encoded
     let token = this.token?.replace(/sig=(.*)/, (...matches) => {
-      return "sig=" + encodeURIComponent(matches[1]);
```

Comment:
Are you sure you don't need this? I believe you do.

Reply:
yup, encode is not needed.